cmd/beacon-chain/usage.go

```diff
@@ -45,154 +45,188 @@ type flagGroup struct {
 }
 
 var appHelpFlagGroups = []flagGroup{
-	{
+	{ // Flags relevant to running the process.
 		Name: "cmd",
 		Flags: []cli.Flag{
-			cmd.MinimalConfigFlag,
+			cmd.AcceptTosFlag,
+			cmd.ConfigFileFlag,
```

Comment:
`cmd.ConfigFileFlag` is here but there are other config flags in `beacon-chain`:
- `cmd.ChainConfigFileFlag` (what's the difference between this one and `cmd.ConfigFileFlag`?)
- `cmd.E2EConfigFlag`
- `flags.ChainID`
- `flags.NetworkID` (what's the difference between this one and `flags.ChainID`?)
- `cmd.MinimalConfigFlag`

Reply:
This doesn't feel right, but I am not entirely sure where they belong. The comment says `Flags relevant to running the process` - which ones of these are relevant?

Reply:
The config file is used for setting all of the flags. 

The other things you mentioned are for configuring the beacon chain so it feels more appropriate to keep them in the beacon chain section.